webrtc.html

```diff
@@ -9005,12 +8994,12 @@ <h3><dfn>RTCDataChannel</dfn></h3>
         </li>
         <li>
           <p>If the <a data-lt="data transport">transport</a> was closed
-          <dfn id="data-transport-closed-error">with an error</dfn>, fire
-          an <code><a>RTCError</a></code> event at <var>channel</var> with
-          <code>errorDetail</code> set to "sctp-failure".</p>
+          <dfn id="data-transport-closed-error">with an error</dfn>, <a>fire
+          an event</a> using the <code><a>RTCError</a></code> interface with
+          <code>errorDetail</code> set to "sctp-failure" at <var>channel</var>.</p>
```

Comment:
Same as above. (And more below.)

Reply:
Also, I'm missing the "named error" part.